packages/office-ui-fabric-react/src/components/__snapshots__/TextField.Placeholder.Example.tsx.shot

```diff
@@ -207,17 +207,8 @@ exports[`Component Examples renders TextField.Placeholder.Example.tsx correctly
         className=
             ms-Label
             {
-<<<<<<< HEAD
```

Comment:
FYI - this diff is correct.. these snapshots were merged with conflicts:

https://github.com/OfficeDev/office-ui-fabric-react/blob/044e13c2c8c391885693bbaa493e975fab35f3ea/packages/office-ui-fabric-react/src/components/__snapshots__/TextField.Placeholder.Example.tsx.shot#L210